sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/README.md

```diff
@@ -58,15 +58,15 @@ The easiest way to create a `ContainerClient` is to use a connection string.
 from azure.storage.blob.aio import ContainerClient
 container_client = ContainerClient.from_connection_string("my_storageacount_connection_string", "mycontainer")
 ```
-For other ways of creating a `ContainerClient`, go to [Blob Storage library](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-blob) for more details.
+For other ways of creating a `ContainerClient`, go to [Blob Storage library](../../storage/azure-storage-blob) for more details.
```

Comment:
This would break PyPI, it's on purpose to have an absolute link when this page is read from somewhere else.

And actually, in this case I would settle it by pointing to https://pypi.org/project/azure-storage-blob/ instead

Reply:
@scbedd can confirm but my understanding is we process these and turn these relative links into absolute links based on the commit sha. I know we do it for docs and we should do it for packaging as well if we aren't already. If we don't then we have potential issues with any moves in master, I don't generally like pointing repo links directly to master as they are guaranteed to be incorrect or broken in the future.

Reply:
Looking at our build pipeline at https://github.com/Azure/azure-sdk-for-python/blob/master/eng/pipelines/templates/steps/build-artifacts.yml#L21-L26 we do run this before we do the packaging so I expect the relative links will be correctly resolved before we package.